x-pack/plugins/alerting/server/rules_client/common/retry_if_bulk_edit_conflicts.test.ts

```diff
@@ -9,6 +9,7 @@ import { KueryNode } from '@kbn/es-query';
 import { retryIfBulkEditConflicts } from './retry_if_bulk_edit_conflicts';
 import { RETRY_IF_CONFLICTS_ATTEMPTS } from './wait_before_next_retry';
 import { loggingSystemMock } from '@kbn/core/server/mocks';
+import { BulkEditSkipReason } from '../../types';
```

Comment:
It's not a path where you can find a type.
I found it here: `import { BulkEditSkipReason } from '../../../common/bulk_edit';`
Not sure why build did not fail.

Reply:
Yes, this **is** weird. 

Thanks for catching, fixed it.